dashboards-observability/public/components/visualizations/charts/helpers/viz_types.ts

```diff
@@ -238,6 +239,14 @@ const getUserConfigs = (
   return isEmpty(configOfUser) ? userSelectedConfigs : configOfUser;
 };
 
+export const getVisTypeData = (vizId: string) => {
+  if (SIMILAR_VIZ_TYPES.includes(vizId)) {
+    return getVisType(vizId, { type: vizId });
+  } else {
```

Comment:
We don't need else here just return 'getVisType(vizId);' as default.

Reply:
done